### PR TITLE
Added code to raise exception in the l1 loss function

### PR DESCRIPTION
**Pull request by Nikhil-Kandukuri**
Raised a value error exception if we have the reduction parameter as sum

### CODE REVIEW

**Comment by albanD** [comment]
Please open PRs on a forked repo, not the main PyTorch repo.
